c10/core/Scalar.h

```diff
@@ -101,16 +105,16 @@ Scalar(T vv)
 };
 
 // define the scalar.to<int64_t>() specializations
-template<typename T>
+template <typename T>
```

Comment:
It would be nice if you could commit clang-format changes separately. This is a big important patch, and it would be nice to not dilute it with whitespace changes.

Reply:
Yeah I didn't expect there were so many whitespace changes, is there a way to undo?

Reply:
I think it should be fine in this case, the added files are more important than changed files for this diff.

Reply:
You could clang-format this entire file in a separate PR and rebase this PR on top, or you can `git reset HEAD^` and then `git add -p` and `git commit` to select only meaningful parts (I hope clang-format hook would not fire on parts you don't touch).

Reply:
Thanks, just updated